package.json

```diff
@@ -57,6 +63,7 @@
   "devDependencies": {
     "@ethersproject/experimental": "^5.4.0",
     "@graphql-codegen/cli": "1.21.5",
+    "@graphql-codegen/schema-ast": "^2.5.1",
     "@graphql-codegen/typescript": "1.22.3",
     "@graphql-codegen/typescript-operations": "^1.18.2",
     "@graphql-codegen/typescript-rtk-query": "^1.1.1",
```

Comment:
It looks like you've removed these from the codegen file--I was surprised to see that. Do we not need `@graphql-codegen/typescript` or  `@graphql-codegen/typescript-operations`?

Reply:
Relay generates the types, so now we only need codegen to grab the schema